integration-tests/hibernate-reactive-mysql-agroal-flyway/pom.xml

```diff
@@ -13,11 +13,6 @@
     <name>Quarkus - Integration Tests - Hibernate Reactive - MySQL - With Agroal and Flyway</name>
     <description>Hibernate Reactive related tests running with the MySQL database and also using Agroal and Flyway</description>
 
-    <properties>
-        <reactive-mysql.url>vertx-reactive:mysql://localhost:3308/hibernate_orm_test?allowPublicKeyRetrieval=true</reactive-mysql.url>
```

Comment:
I'm surprised that we're losing the `allowPublicKeyRetrieval` and not reintroducing it anywhere... was it pointless?